### PR TITLE
MODCONF-96 Config data settings naming migration script

### PR DESCRIPTION
**Pull request by Mykyta-Varenyk**
Resolves: https://issues.folio.org/browse/MODCONF-96.
### Problem
After upgrade from Iris to Juniper, all checks in Settings for Circulation -> Other Settings -> Patron id(s) section disappeared. It has been discovered that the issue appeared due to the new settings format that frontend sends to the backend, and so records in both old and new format are now stored in the database. 
### Approach
The decision has been made to fix this issue with the data migration.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-configuration</artifactId>
-  <version>5.8.0-SNAPSHOT</version>
+  <version>5.8.1-SNAPSHOT</version>
```
Comment (adamdickmeiss) [inline pom.xml:5]: Version 5.8.0 is the next version and has not been released. Why are you changing that version?

File: mod-configuration-server/src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -4,6 +4,11 @@
       "run": "after",
       "snippetPath": "migration/config_data.sql",
       "fromModuleVersion": "mod-configuration-5.3.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migration/migrateConfigDataOtherSettingsNamings.sql",
+      "fromModuleVersion": "mod-configuration-5.8.1"
```
Comment (adamdickmeiss) [inline mod-configuration-server/src/main/resources/templates/db_scripts/schema.json:11]: Should simply be 5.8.0 , so it's triggered from all versions before 5.8.0. eg 5.7.1.

Comment (Mykyta-Varenyk) [reply mod-configuration-server/src/main/resources/templates/db_scripts/schema.json]: Fixed

File: pom.xml (hunk 1 above)
Comment (Mykyta-Varenyk) [reply pom.xml]: Fixed

**Review by adamdickmeiss — APPROVED** [review]
Thx

**Review by alexanderkurash — APPROVED** [review]

**Pull request merged**